src/main/java/parser/Parser.java

```diff
@@ -34,6 +34,8 @@ public static Command parse(String fullCommand, Access access)
         String commandType = commandTypeAndArgs[0].trim().toLowerCase();
         String commandArgs = commandTypeAndArgs[1].trim();
 
+        System.out.println("Command Type: " + commandType);
```

Comment:
Perhaps could use ui.showToUser as well. And could maybe change to "Command entered: " instead? As some commands may be invalid.